spec/lucky/link_helpers_spec.cr

```diff
@@ -45,6 +45,36 @@ private class TestPage
       text "Hello"
     end
   end
+
+  def string_path_with_anchor
+    link "Hello", to: "/foo", anchor: "bar"
+  end
+
+  def string_path_with_anchor_and_hashmark
+    link "Hello", to: "/foo", anchor: "#bar"
+  end
+
+  def get_route_with_anchor
+    link "Test", to: LinkHelpers::Index, anchor: "bar"
+  end
+
+  def non_get_route_with_anchor
+    link "Test", to: LinkHelpers::Create, anchor: "bar"
+  end
+
+  def non_get_route_with_options_with_anchor
+    link "Test", to: LinkHelpers::Create, something_custom: "foo", anchor: "bar"
+  end
+
+  def string_path_with_options_with_anchor
+    link "Test", to: "/foos", data_method: "post", anchor: "bar"
+  end
+
+  def get_route_with_block_with_anchor
```

Comment:
I don't the link helper need to be modified because you can do `link "Test", to: LinkHelpers::Create.with(anchor: "bar")`. This will keep the link helper simpler and makes it so there is just one way to add an anchor